PowerPointLabs/PowerPointLabs/Utils/GraphicsUtil.cs

```diff
@@ -191,6 +192,51 @@ private static void DeleteSpecificFilePath(string filePath)
         #endregion
 
         #region Slide
+        public static void ExportSlides(List<Slide> slides, string exportPath, float magnifyRatio = 1.0f)
+        {
+            if (slides.Count <= 0)
+            {
+                return;
+            } 
+            else if (slides.Count == 1)
+            {
+                ExportSlide(slides[0], exportPath, magnifyRatio);
+                return;
+            }
+
+            if (!Directory.Exists(exportPath))
+            {
+                // Remove the default ".png" extension from folder name
+                string folderName = exportPath.Substring(0, exportPath.Length - 4);
+
+                try
+                {
+                    Directory.CreateDirectory(folderName);
+
+                    // Alert the user that the slides will be saved in a folder
+                    string messageBoxText = "Each slide in your presentation has been saved as a separate file in the folder " + folderName + ".";
+                    MessageBox.Show(messageBoxText);
+
+                    foreach (Slide slide in slides)
+                    {
+                        string fileName = folderName + "\\" + slide.Name + ".png";
+                        ExportSlide(slide, fileName, magnifyRatio);
+                    }
+                }
+                catch (Exception)
+                {
+
+                    // Failed to create directory, we save the images all to the specified path
+                    foreach (Slide slide in slides)
```

Comment:
Will there be a message displayed for failure to create directory?

Reply:
Added a message box!